cmd/sonobuoy/app/retrieve.go

```diff
@@ -85,7 +85,7 @@ func retrieveResults(cmd *cobra.Command, args []string) {
 
 	err = eg.Wait()
 	if _, ok := err.(exec.CodeExitError); ok {
-		fmt.Fprintln(os.Stderr, "Results not ready yet. Check `sonobuoy status` for status.")
+		fmt.Fprintln(os.Stderr, "Results not ready yet at /tmp/sonobuoy/. Check `sonobuoy status` for status.")
```

Comment:
Well... that might be confusing, and it's customizable.

Reply:
Can I add parameters there ? the customized path you show there in stderr. what do you suggest ?

Reply:
There is a config variable that is used to indicate output location.  

https://github.com/heptio/sonobuoy/blob/master/pkg/config/config.go#L276